gwr-track/src/perfetto_trace_builder.rs

```diff
@@ -70,6 +70,20 @@ impl PerfettoTraceBuilder {
         track_descriptor
     }
 
+    fn build_value_track_descriptor(&mut self, id: Id, parent: Id, name: &str) -> TrackDescriptor {
```

Comment:
I think we're at risk of having two naming conventions cohabiting in this `impl` block at the moment though this may not really matter (previously the function names related solely to the names used in the Perfetto schema, and not the names used elsewhere within gwr-track).
What's your feeling on whether it would make more sense to rename the "counter_track" functions in terms of enter/exit events or the "value_track" functions in terms of being a non-incremental double counter? I'm currently on the fence on this.